openapi/tea_spec_v0.3.0.yaml

```diff
@@ -0,0 +1,527 @@
+openapi: 3.0.3
+info:
+  title: Transparency Exchange API Specification
+  description: |-
+    This is an OpenAPI specification of the Transparency Exchange API (TEA) - Project Koala.
+    
+    The Transparency Exchange API allows for the discovery, retrieval, and publication of software transparency artifacts such as SBOMs, VEX, and other related documents.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.3.0
+servers:
+  - url: https://tea.example.com/0.3.0/
+    description: Production Transparency Exchange API
+paths:
+  /.well-known/tei/{tei}:
```

Comment:
I do not believe that this endpoint makes a lot of sense. 
If somebody finds this endpoint they would already have found the TEA api and do not require the well-known endpoint anymore.

Maybe I am understanding that wrong, but I wouldn't have put that endpoint on the TEA service but rather on some top level domain e.g. <company>.com/.well-known/tea. Otherwise this would be a bit of a chicken and egg problem wouldn't it?

Reply:
You are commenting on a not-up-to-date repo. The current docs are pointing to /.well-known/tea to get a 302 redirect, since we're skipping the URI DNS records we need to find the API on a specific host.